### PR TITLE
Feature/88 curated solutions

### PR DESCRIPTION
**Pull request by 0qy**

### CODE REVIEW

File: src/pages/MySolutions.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  }, [currentImage]);
+
+  return (
+    <>
+      <Typography variant="h5" gutterBottom>
+        My Solutions
+      </Typography>
+      <ControlsBar>
+        <Button variant="contained" color="default" onClick={() => setShowList((v) => !v)}>
+          Show Report
```
Comment (chrisbc) [inline src/pages/MySolutions.tsx:58]: @0qy on GT view this button changes Text between "Show Report" & "Show List" but not here.

File: src/components/common/ControlsBar.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    alignItems: 'center',
+  },
+  control: {
+    margin: 10,
+  },
+}));
+interface ControlsBarProps {
+  children: React.ReactNode;
+}
+const ControlsBar: React.FC<ControlsBarProps> = ({ children }: ControlsBarProps) => {
```
Comment (chrisbc) [inline src/components/common/ControlsBar.tsx:16]: @0qy this class could possibly be replaced with MUI Toolbar (). NB the MUI examples aren't great but [this article looks helpful ](https://stackoverflow.com/questions/52653103/what-is-appbar-vs-toolbar)

**Review by chrisbc — CHANGES_REQUESTED** [review]
Thanks @0qy I  picked up 2 things for you to look at but overall very tidy thanks :)

File: src/components/common/ControlsBar.tsx (hunk 2 above)
Comment (0qy) [reply src/components/common/ControlsBar.tsx:16]: Just tried out the Toolbar works the same so I will go ahead and delete that component.

**Review by wooneh — APPROVED** [review]

**Comment by wooneh** [comment]
Looks good to me! Just catching up and it looks great.

**Review by chrisbc — APPROVED** [review]

**Pull request merged**